src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
 body {
   margin: 0;
+  padding: 0;
 }
+
+@import '/src/utils/variables';
+@import '/src/styles/header';
+@import '/src/styles/main';
+@import '/src/styles/card';
```

Comment:
Ensure that the path to the card styles file is correct. It should be relative to the current file location.